var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -36,18 +36,19 @@ class PyNumpy(PythonPackage):
     homepage = "http://www.numpy.org/"
     url      = "https://pypi.io/packages/source/n/numpy/numpy-1.9.1.tar.gz"
 
+    version('1.12.0', '33e5a84579f31829bbbba084fe0a4300',
+            url="https://pypi.io/packages/source/n/numpy/numpy-1.12.0.zip")
```

Comment:
I believe the extrapolator (at least my one) will base the URL on the most recent URL in the list. So maybe there is no need? It's generally good not to do this too frequently, as it means people will need to re-download (and maybe re-sync mirrors) but I don't have a problem with doing it every once in a while for consistency.